### PR TITLE
Disable test in System.Net.Sockets for UAP

### PR DESCRIPTION
**Pull request by caesar1995**
Now System.Net.Sockets.Functional tests will run clean in both innerloop and outerloop for UAP.

I added the [Fact] label back for MulticastInterface_Set_AnyInterface_Succeeds(). The test will never run with only [Outerloop] label. See details here: https://github.com/dotnet/corefx/commit/6257f450653381af91ca5ff097dab837426e91ef#commitcomment-22500352

Contributes to: #20135

### CODE REVIEW

File: src/System.Net.Sockets/tests/FunctionalTests/LoggingTest.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace System.Net.Sockets.Tests
     public class LoggingTest : RemoteExecutorTestBase
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.Uap, "NetEventSource is only part of .NET Core and requires internal Reflection.")]
```
Comment (davidsh) [inline src/System.Net.Sockets/tests/FunctionalTests/LoggingTest.cs]: I think the SkipOnTargetFramework attribute should be left alone.  This test will never run on .NET Framework.

However, skipping the test on Uap is temporary and will be solved as part of getting NetEventSource working on Uap. So, for now, you should add an ActiveIssue attribute for that and leave the SkipOnTargetFramework attribute alone.

Comment applies to similar changes below.

Comment (caesar1995) [inline src/System.Net.Sockets/tests/FunctionalTests/LoggingTest.cs]: ok, will fix

**Comment by caesar1995** [comment]
@davidsh Do I need to run Outerloop tests for my changes in this PR?

**Comment by davidsh** [comment]
>@davidsh Do I need to run Outerloop tests for my changes in this PR?

Normally, no, because you only disabled tests.

But you also added back a Fact attribute which means you added a new test to Outerloop

>public async Task MulticastInterface_Set_AnyInterface_Succeeds()

So, you should run Outerloop for Windows and *Nix on this PR.

**Comment by caesar1995** [comment]
@dotnet-bot help

**Comment by caesar1995** [comment]
@dotnet-bot help

**Comment by caesar1995** [comment]
@dotnet-bot test outerloop netcoreapp Windows 7 Release
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netfx Windows_NT Debug
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug
@dotnet-bot test outerloop netcoreapp Ubuntu16.10 Debug

**Comment by caesar1995** [comment]
@dotnet-bot test Innerloop Tizen armel Release Cross Build (The build timed out)
@dotnet-bot test OuterLoop netcoreapp OSX10.12 Debug x64 (The plugin hasn't been performed correctly: Problem on deletion)

**Comment by danmosemsft** [comment]
@dotnet-bot test OuterLoop netcoreapp OSX10.12 Debug x64 (The plugin hasn't been performed correctly: Problem on deletion)

**Comment by danmosemsft** [comment]
@dotnet-bot test portable linux x64 release build (https://github.com/dotnet/corefx/issues/20441)

**Pull request merged**